integrations/tensorflow/compiler/dialect/tf_tensorlist/conversion/test/convert_tf_to_tf_tensorlist.mlir

```diff
@@ -27,17 +27,18 @@ func @stack(%arg0: tensor<?xf32>, %element_shape: tensor<0xi32>) -> tensor<?xf32
 }
 
 // CHECK-LABEL: func @concat
-func @concat(%arg0: tensor<?xf32>, %element_shape: tensor<0xi32>, %lead: tensor<i64>) -> (tensor<?xf32>, tensor<0xi64>) {
-  // CHECK-DAG: [[LIST0:%.+]] = "tf_tensorlist.FromTensor"(%arg0, %arg1)
+func @concat(%arg0: tensor<5xf32>, %element_shape: tensor<0xi32>, %lead: tensor<i64>) -> (tensor<5xf32>, tensor<0xi64>) {
```

Comment:
why change `?xf32` to `5xf32`?

Reply:
It let me run both passes in sequence for testing. I will revert back to dynamic.